audited.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency 'appraisal'
   gem.add_development_dependency 'rails', '>= 4.2', '< 5.2'
+  gem.add_development_dependency 'rubocop', '~> 0.54.0'
```

Comment:
needs to be locked down to the minor since new cops gets added all the times and then would break the tests